tools/log4j2.xml

```diff
@@ -36,7 +36,7 @@
                      ignoreExceptions="false">
             <ThresholdFilter level="debug"/>
             <PatternLayout>
-                <Pattern>%d{yyyyMMdd HH:mm:ss} %m%n</Pattern>
+                <Pattern>%d{yyyy-MM-dd'T'HH:mm:ss'Z'} %m%n</Pattern>
```

Comment:
Don't change the date/time format in the whois logs, as it breaks any parsing code.

Reply:
reverted